src/site/pages/digital/src/containers/SelectionPopup/modules/ColorModule.tsx

```diff
@@ -27,4 +27,4 @@ export const ColorModule = PopupModule({
         config: Config,
         alt: "Color of object(s)"
     })]
-});
+}) as (props: UseModuleProps) => JSX.Element;
```

Comment:
This all the others should be resolved some other way, presumably by just changing `PopupModule` function return type or whatever

Reply:
`PopupModule` can return `null` looking at it closer though. If all the passed in modules are null, then it returns null.